extensions/pyRevitTools.extension/pyRevit.tab/Project.panel/ptools.stack/Links.pulldown/ListDWGs.pushbutton/ListDWGs_script.py

```diff
@@ -33,34 +37,54 @@ def listdwgs(current_view_only=False):
         output.print_md("####{}".format(link_mode))
         for dwg in dwgInst[link_mode]:
             dwg_id = dwg.Id
-            dwg_name = \
-                dwg.Parameter[DB.BuiltInParameter.IMPORT_SYMBOL_NAME].AsString()
+            dwg_name = dwg.Parameter[DB.BuiltInParameter.IMPORT_SYMBOL_NAME].AsString()
             dwg_workset = revit.query.get_element_workset(dwg).Name
-            dwg_instance_creator = \
-                DB.WorksharingUtils.GetWorksharingTooltipInfo(revit.doc,
-                                                              dwg.Id).Creator
+            dwg_instance_creator = DB.WorksharingUtils.GetWorksharingTooltipInfo(
+                revit.doc, dwg.Id
+            ).Creator
 
-            if current_view_only \
-                    and revit.active_view.Id != dwg.OwnerViewId:
+            if current_view_only and revit.active_view.Id != dwg.OwnerViewId:
                 continue
 
-            print('\n\n')
-            output.print_md("**DWG name:** {}\n\n"
-                            "- DWG created by:{}\n\n"
-                            "- DWG id: {}\n\n"
-                            "- DWG workset: {}\n\n"
-                            .format(dwg_name,
-                                    dwg_instance_creator,
-                                    output.linkify(dwg_id),
-                                    dwg_workset))
-
-
-selected_option = \
-    forms.CommandSwitchWindow.show(
-        ['In Current View',
-         'In Model'],
-        message='Select search option:'
-        )
+            # Get hosted level name
+            level = revit.doc.GetElement(dwg.LevelId)
+            level_name = level.Name if level else "N/A"
+
+            # Get offset using the Transform
+            offset_z = "N/A"
+            offset_z_m = "N/A"
+            try:
+                transform = dwg.GetTransform()
+                if transform:
+                    offset_z = transform.Origin.Z  # in feet
+                    offset_z_m = round(
+                        offset_z * 0.3048, 3
```

Comment:
Consider using ConvertFromInternalUnits Method instead of the multiplicator
https://apidocs.co/apps/revit/2022/60c6aac3-8306-c56e-b62f-b7011b9ad7b6.htm#